astropy/tests/helper.py

```diff
@@ -304,14 +304,27 @@ def _save_coverage(cov, result, rootdir, testing_path):
     # Python 2.x.
```

Comment:
FYI: there's almost no affiliated packages that use 2to3 any more:
https://github.com/astropy/package-template/pull/93#issuecomment-73303102